### DISCUSSION TITLE
Klinger Volume Oscillator

### DISCUSSION THREAD

**Discussion by DaveSkender** [DaveSkender/Stock.Indicators#446] [Indicators]
Community discussion about [Klinger Volume Oscillator](https://dotnet.stockindicators.dev/indicators/Kvo/#content)

**Comment by DaveSkender** [comment]
Stephen Klinger's description: [Klinger Volume Oscillator](http://download.mtachn.com/upload/article/201608171328304062291.pdf)

**Comment by August1328** [comment]
Hi Dave,

is there a reason why the function for the KVO is called get_keltner() and not e.g. get_klinger()? See here: https://python.stockindicators.dev/indicators/Kvo/#content

Thanks,
Andy

**Reply by DaveSkender** [reply]
No.  It looks like a typo in the documentation site.  Code looks okay.  Thanks for reporting!
Are you interested in contributing a fix?

**Reply by August1328** [reply]
Thanks for the quick reply.
Unfortunately I don´t know how to contribute a fix, I still have to learn how Github works, besides posting questions and comments...

**Reply by DaveSkender** [reply]
Not a problem.  I just like to give people the opportunity to contribute.  Finding and reporting are also appreciated contributions.  You can track this issue here: https://github.com/DaveSkender/Stock.Indicators.Python/issues/227
